expat/configure.ac

```diff
@@ -222,6 +222,17 @@ AC_DEFINE([XML_DTD], 1,
 AC_DEFINE([XML_DEV_URANDOM], 1,
           [Define to include code reading entropy from `/dev/urandom'.])
 
+AC_ARG_ENABLE([xml-attr-info],
+  [AS_HELP_STRING([--enable-xml-attr-info],
+                 [Enable retrieving the byte offsets for attribute names and values])], ,
+                 [enable_xml_attr_info=yes])
```

Comment:
Please turn default to `no` and mention the default in the help text.

Reply:
done, set XML_ATTR_INFO disable by default.